lms/models/public_id.py

```diff
@@ -48,15 +48,13 @@ def generate_instance_id(cls) -> str:
     def parse(
         cls,
         public_id: str,
-        expect_app_code: str | None = "lms",
-        expect_model_code: str | None = None,
-        expect_region: Region | None = None,
+        expect_model_code: str,
+        expect_region: Region,
```

Comment:
The `expect_app_code` argument was never used, and the `expect_model_code` and `expect_region` arguments are always used. This method is only called in one place and it passes `expect_model_code` and `expect_region` but does not pass `expect_app_code`.

This is a weird method. It takes a `public_id` string and parses it to return a `Region`, but you have to pass in the exact `Region` that you expect it to return as the `expect_region` argument and it raises if the region that it's going to return is not equal to `expect_region`.

If I already know what the method is going to return, then why call it?